src/main/scala/uk/gov/ons/bi/dataload/exports/HmrcBiCsvExtractor.scala

```diff
@@ -31,17 +31,23 @@ object HmrcBiCsvExtractor {
 
     // Extract data from main data frame
 
-    def stringify(stringArr: Column) = concat(lit("["), concat_ws(",", stringArr), lit("]"))
+    def stringifyArr(stringArr: Column) = concat(lit("["), concat_ws(",", stringArr), lit("]"))
 
     def getHMRCOutput(df: DataFrame): DataFrame = {
-      df.withColumn("arrVar", df("VatRefs").cast(ArrayType(StringType)))
-        .withColumn("arrPaye", df("PayeRefs").cast(ArrayType(StringType)))
-        .withColumn("VatRef", stringify($"arrVar"))
-        .withColumn("PayeRef", stringify($"arrPaye"))
-        .select("id","BusinessName","TradingStyle","PostCode",
-          "Address1", "Address2","Address3","Address4", "Address5",
-          "IndustryCode","LegalStatus","TradingStatus",
-          "Turnover","EmploymentBands","CompanyNo","VatRef","PayeRef")
+        val arrDF = df
+          .withColumn("arrVar", df("VatRefs").cast(ArrayType(StringType)))
+          .withColumn("arrPaye", df("PayeRefs").cast(ArrayType(StringType)))
+
+        val dropDF = arrDF
+          .withColumn("VatRef", stringifyArr(arrDF("arrVar")))
+          .withColumn("PayeRef", stringifyArr(arrDF("arrPaye")))
+          .drop("VatRefs","PayeRefs","arrVar", "arrPaye")
+
+        dropDF
```

Comment:
These are magic strings, they should be held in an object somewhere, see the [business-index-api](https://github.com/ONSdigital/business-index-api/blob/master/app/models/BusinessFields.scala)